### PR TITLE
Correction of various small typos

### PR DESCRIPTION
**Pull request by jpellman**

### CODE REVIEW

File: system/python.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ next:
 
 # Python
 
-I know that we said earlier that you don't need to be able to write Python or other code to use Salt, which is true. However, it is useful in many circumstances to be able to *read* Python, or at least Python function documentation, which turns out is much easier that writing Python.
+I know that we said earlier that you don't need to be able to write Python or other code to use Salt, which is true. However, it is useful in many circumstances to be able to *read* Python, or at least Python function documentation, which turns out is much easier than writing Python.
```
Comment (teranth) [inline system/python.md]: Ah good, someone else saw it, no need for a PR heh

**Comment by jpellman** [comment]
@cachedout - Okay to merge PR?

**Comment by cachedout** [comment]
@jpellman Same situation here as the other issue. cc: @sagetherage to find out who is maintaining this now.

**Comment by sagetherage** [comment]
@newwebash @s0undt3ch this also looks good to me, can one of you review, today and I will work on this repo's maintainer ownership within SaltStack - @newwebash likely that will be you until decided otherwise.

**Pull request merged**